sample/Saturn.Sample/Saturn.Sample.fs

```diff
@@ -2,6 +2,7 @@ module Sample
 
 open Saturn
 open Giraffe
+open Microsoft.AspNetCore.Http
```

Comment:
Changes in this file are not really used any more, right?

Reply:
Will revert, thanks for noticing!